src/sentry/web/frontend/auth_login.py

```diff
@@ -197,7 +197,7 @@ def post(self, request: Request, **kwargs) -> HttpResponse:
         if op == "sso" and request.POST.get("organization"):
             return self.redirect_post_to_sso(request=request)
 
-        organization = kwargs.pop("organization", None)
+        organization: RpcOrganization = kwargs.pop("organization", None)
```

Comment:
```suggestion
        organization: RpcOrganization | None = kwargs.pop("organization", None)
```

Couldn't this be none as well?